os/Kconfig

```diff
@@ -279,6 +279,13 @@ config BOARD_BUILD_DATE
 		It is included in binary header as binary version and binary manager uses it to check the latest binary.
 		So it should be set to a higer value than versions of old binaries if you want to make the binary the latest version.
 
+config BINARY_SIGNING
+	bool "Enable Binary Signing"
+	default n
+	depends on ARCH_BOARD_RTL8721CSM
+	---help---
+	Enable RDP encryption and Secure boot.
```

Comment:
Encryption algorithm can be different for each board.
So I think just "Enable Encryption and Secure boot" is better.